whipper/command/mblookup.py

```diff
@@ -12,35 +14,60 @@ class MBLookup(BaseCommand):
 
     def add_arguments(self):
         self.parser.add_argument(
-            'mbdiscid', action='store', help="MB disc id to look up"
+            'id', action='store', help="MB disc id or release id to look up"
         )
 
+    def _printMetadata(self, md):
+        """
+        Print out metadata received in a sensible way
+
+        :param md: MusicBrainz's metadata about the disc
+        :type md: `DiscMetadata`
+        rtype: None
+        """
+        print('    Artist: %s' % md.artist.encode('utf-8'))
+        print('    Title:  %s' % md.title.encode('utf-8'))
+        print('    Type:   %s' % str(md.releaseType).encode('utf-8'))
+        print('    URL:    %s' % md.url)
+        print('    Tracks: %d' % len(md.tracks))
+        if md.catalogNumber:
+            print('    Cat no: %s' % md.catalogNumber)
+        if md.barcode:
+            print('    Barcode: %s' % md.barcode)
+
+            for j, track in enumerate(md.tracks):
+                print('      Track %2d: %s - %s' % (
+                    j + 1, track.artist.encode('utf-8'),
+                    track.title.encode('utf-8')
+                ))
+        return None
```

Comment:
Isn't this return useless? (the default return value is already `None`).
If you remove this instruction I would suggest removing the `rtype` entry in the docstring too.